src/utils/requests.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const PIZZERIA_HOST = '';
```

Comment:
How do you override this?

Reply:
Ideally, I can override it by changing the environment am using from development to production, which means the ternary conditional operator is modified to suit my needs.